nixos/modules/system/boot/uvesafb.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.boot.uvesafb;
```

Comment:
A hint:

Change all `lib.<X>` to `X` and  list them here as `inherit (lib) X ...;`